src/containers/Programs/index.tsx

```diff
@@ -248,7 +248,7 @@ const Programs = ({
                 </FiltersSearchWrapper>
               </div>
               <div>
-                {stateToRender.map((data, index) => {
+                {(!sortedInfo.length ? stateToRender : sortedInfo).map((data, index) => {
```

Comment:
How will that work with favourites/ search /filter? 
Even if we are using a separate array for favourites we still assign it to `stateToRender` before we do .map. So regardless if it is a filter or search or favourites, it all goes via one state: `stateToRender`. But if you will be using` sortedInfo` to map instead it will make search and filter and favourites stop working?